components/SpotlightBanner/__tests__/SpotlightBannerAside.spec.tsx

```diff
@@ -8,14 +8,14 @@ import { SpotlightBannerAside } from '../SpotlightBannerAside';
 
 describe('SpotlightBannerAside', () => {
   test('renders correctly', () => {
-    const renderer = TestRenderer.create(<SpotlightBannerAside />).toJSON();
+    const renderer = TestRenderer.create(<SpotlightBannerAside classname={'spotlight__comparison'} />).toJSON();
```

Comment:
Like I said before, the purpose of unit tests is to catch changes that may break things. This was clearly a breaking change since u completely changed the classes.